src/MCPServer/requirements/base.txt

```diff
@@ -0,0 +1,7 @@
+Django>=1.7,<1.8
+django-mysqlpool==0.1-9
+django-extensions==1.1.1
+mysqlclient==1.3.7
+gearman==2.0.2
+lxml==3.5.0
+pyinotify
```

Comment:
I don't think you are correct, @sevein .  In stable/1.5.x pyinotify is still used:

https://github.com/artefactual/archivematica/blob/stable/1.5.x/src/MCPServer/lib/archivematicaMCP.py#L40

It is not used in qa/1.x, but this PR is for stable/1.5.x, so pyinotify is still needed.